### ISSUE TITLE
cargo compiles 0.108.0 instead of 0.109.0 on Mac Intel

### ISSUE THREAD

**Issue by ondohotola** [jqnatividad/qsv#1152]
As I need `excel, lua, to, sqlp` I ran

```bash
cargo install qsv --locked --features apply,generate,luau,fetch,foreach,to,polars,feature_capable
```

which installs all the above but in v 0.108.0.

What must I do on the Intel Mac to get 0.109.0?

**Comment by jqnatividad** [comment]
Hi @ondohotola ,
This is because I didn't publish 0.109.0 to crates.io.

I decided to hold off publishing as qsv 0.109.0 has outgrown Github's free Windows Action Runners with out-of-memory errors. So I'll have to figure out a way to build  the Windows binaries with a self-hosted runner similar to how the Apple Silicon binaries are built.

Also, 0.109.1 has fixes for describegpt and join, and I'm waiting on Homebrew to upgrade Rust to 1.71.0.

I only found out about the Windows Action Runner problem when I released 0.109.0 - and that's when I held off publishing to crates.io.

I'm targeting to release 0.109.1 this weekend.  In the meantime, you can build 0.109.0 from source by using cargo build:

```
git clone --branch 0.109.0 https://github.com/jqnatividad/qsv.git
cargo build qsv --locked --features apply,generate,luau,fetch,foreach,to,polars,feature_capable
```

**Comment by ondohotola** [comment]
Thanks, rust has been updated by Homebrew and it compiles
